packages/ui/src/views/components/RadialSelect.tsx

```diff
@@ -9,21 +9,21 @@ export interface RadialOption<T extends string> {
   value: T
 }
 
-interface RadialInputProps<T extends string> {
+interface RadialSelectProps<T extends string> {
   options: RadialOption<T>[]
   value: T
   onValueChange: (value: T) => void
   id?: string
   className?: string
 }
 
-export const RadialInput = <T extends string>({
+export const RadialSelect = <T extends string>({
```

Comment:
This probably shouldn't be called `Radial` since the words "Radio" and "Radial" have nothing to do with each other. Probably should be `RadioSelect`